### ISSUE TITLE
Derived attributes were directly below 'normal' attributes. 

### ISSUE THREAD

**Issue by theoryshaw** [buildingSMART/IFC4.4.x-development#19]
In the UI, would be nice if derived attributes were directly below 'normal' attributes. 
Example below...
![image](https://user-images.githubusercontent.com/507113/184550319-1c8fc7bb-1ed0-4922-b6db-7f71049e596f.png)

**Comment by aothms** [comment]
I agree, the issue though is that the derived attributes are defined in subtypes of the classes where the original attributes are derived (which are collapsed by default). Maybe instead of moving the derived attributes up, the original attributes should be moved down in case of derivations in subtypes?

**Comment by theoryshaw** [comment]
Sorry, don't know enough to know. :)  
Is there an example you could provide?
